tests/cleaner_tests/ipv6_test/ipv6_test.py

```diff
@@ -8,7 +8,7 @@
 
 from sos_tests import StageTwoReportTest
 
-MOCK_FILE = '/tmp/sos-test-ipv6.txt'
+MOCK_FILE = '/sos-test-ipv6.txt'
```

Comment:
This change (and same in `ipv6_test.py` is tricky. Some previous test obfuscated `tmp` into `obfuscatedword1`. We replaced `/etc/sos/cleaner/default_mapping` by empty file, yet the cleaner's cache kept the mapping. So `self.assertFileCollected(MOCK_FILE)` fails as the filename is changed by cleaner in the tarball.